cmd2/cmd2.py

```diff
@@ -2865,14 +2886,20 @@ def do_set(self, args: argparse.Namespace) -> None:
             if onchange_hook is not None:
                 onchange_hook(old=current_value, new=value)
```

Comment:
I don't think this line is covered.  If it isn't too difficult, it would be nice to cover.